app/modules/ps/index.ts

```diff
@@ -80,14 +81,29 @@ const PSModule: ModuleFunction = async () => {
 
   server.add('/champ/:id', async ({ params, payload }) => {
     const { id } = params;
-    const { laneId, tierId } = payload;
+    let { laneId, rankRangeId = 2 } = payload;
 
-    const {
-      data: { data: champArguments },
-    } = await axios.get(`https://lol.ps/api/champ/${id}/arguments.json`);
+    // 선택한 라인이 없다면 챔피언의 기본 라인을 가져오기
+    if (laneId === null || laneId === undefined) {
```

Comment:
```suggestion
    if (isNil(laneId)) {
```
https://lodash.com/docs/4.17.15#isNil
쓰는 건 어떠신가요?